DataFormats/TrackerCommon/src/TrackerTopology.cc

```diff
@@ -85,6 +85,29 @@ unsigned int TrackerTopology::module(const DetId &id) const {
   return 0;
 }
 
+std::pair<int,int> TrackerTopology::stripSubDetAndLayer(const DetId& id) const
```

Comment:
Good point... actually, after a closer look, I don't think this method belongs in `TrackerTopology` at all (the 'layer' number differs by 1 in with `TrackerTopology::layer`, and ring instead of wheel for TID and TEC) - the numbering scheme it assumes seems to be only used for the parameters passed to a few of the fake ESSources, so I moved it back into CalibTracker/SiStripESProducers (and made a helper class to share the code between the three plugins that use it)